### DISCUSSION TITLE
Search for heavy spike-in?

### DISCUSSION THREAD

**Discussion by ht-lau** [vdemichev/DiaNN#315] [General]
Hi Vadim,
Is there any way to search for heavy peptides only from my spike-in,  like the Thermo retention time standard?  Can I add the modifications to the fasta?

Thanks in advance.  
HT

**Comment by vdemichev** [comment]
Yes, you can search for spike ins and add arbitrary modifications, see --var-mod and --fixed-mod commands description in the docs. The next version to be made public soon will have signficantly improived SILAC support.
